render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/JpqlRenderContext.kt

```diff
@@ -370,6 +371,7 @@ private class DefaultModule : JpqlRenderModule {
             JpqlPredicateParenthesesSerializer(),
             JpqlRoundSerializer(),
             JpqlSelectQuerySerializer(),
+            JpqlPowerSerializer(),
```

Comment:
Could you please move the position of `JpqlPowerSerializer()` below `JpqlPlusSerializer()`?

Reply:
You mean alphabetical order! Got it :)